apps/studio/components/interfaces/HomeNew/ActivityStats.tsx

```diff
@@ -0,0 +1,131 @@
+import dayjs from 'dayjs'
+import { GitBranch } from 'lucide-react'
+import Link from 'next/link'
+import { useMemo } from 'react'
+
+import { useParams } from 'common'
+import { useBranchesQuery } from 'data/branches/branches-query'
+import { useBackupsQuery } from 'data/database/backups-query'
+import { useMigrationsQuery } from 'data/database/migrations-query'
+import { useSelectedProjectQuery } from 'hooks/misc/useSelectedProject'
+import { cn, Skeleton } from 'ui'
+import { TimestampInfo } from 'ui-patterns'
+import { ServiceStatus } from './ServiceStatus'
+
+export const ActivityStats = () => {
+  const { ref } = useParams()
+  const { data: project } = useSelectedProjectQuery()
+
+  const { data: branchesData, isLoading: isLoadingBranches } = useBranchesQuery({
+    projectRef: project?.parent_project_ref ?? project?.ref,
+  })
+  const isDefaultProject = project?.parent_project_ref === undefined
+  const currentBranch = useMemo(
+    () => (branchesData ?? []).find((b) => b.project_ref === ref),
+    [branchesData, ref]
+  )
+  const latestNonDefaultBranch = useMemo(() => {
+    const list = (branchesData ?? []).filter((b) => !b.is_default)
+    if (list.length === 0) return undefined
+    return list
+      .slice()
+      .sort(
+        (a, b) =>
+          new Date(b.created_at ?? b.updated_at).valueOf() -
+          new Date(a.created_at ?? a.updated_at).valueOf()
+      )[0]
+  }, [branchesData])
+
+  const { data: migrationsData, isLoading: isLoadingMigrations } = useMigrationsQuery({
+    projectRef: project?.ref,
+    connectionString: project?.connectionString,
+  })
+  const latestMigration = useMemo(() => (migrationsData ?? [])[0], [migrationsData])
+
+  const { data: backupsData, isLoading: isLoadingBackups } = useBackupsQuery({
+    projectRef: project?.ref,
+  })
+  const latestBackup = useMemo(() => {
+    const list = backupsData?.backups ?? []
+    if (list.length === 0) return undefined
+    return list
+      .slice()
+      .sort((a, b) => new Date(b.inserted_at).valueOf() - new Date(a.inserted_at).valueOf())[0]
+  }, [backupsData])
+
+  return (
+    <div className="@container">
+      <div className="grid grid-cols-2 xl:flex xl:gap-10 gap-4 flex-wrap">
+        <div className="block">
+          <p className="heading-meta text-foreground-light mb-1">Status</p>
+          <ServiceStatus />
+        </div>
+
+        <Link className="block" href={`/project/${ref}/database/migrations`}>
```

Comment:
Might be nice to add a little hover state on the children of these like:

```jsx
<Link className="block hover:[&_div>p]:text-foreground transition-colors" />
```

...so their interactivity is reinforced